src/mute/muteReducers.js

```diff
@@ -0,0 +1,14 @@
+import {
+  REALM_INIT,
+} from '../constants';
+
+const initialState = [];
+
+export default (state = initialState, action) => {
+  switch (action.type) {
+    case REALM_INIT:
```

Comment:
I'm not sure this should be its own reducer though (maybe something like `state.settings` ...?) or at least we might want to rename it `state.mutedTopics` to be clearer